packages/react-components/react-carousel-preview/library/src/components/Carousel/Carousel.types.ts

```diff
@@ -45,3 +49,9 @@ export interface CarouselVisibilityEventDetail {
 }
 
 export type CarouselVisibilityChangeEvent = CustomEvent<CarouselVisibilityEventDetail>;
+
+export interface CarouselReinitData {
```

Comment:
```suggestion
/**
 * @internal
 */
export interface CarouselReinitData {
```

Let's ensure that it won't be exported